src/languages/r.js

```diff
@@ -2,76 +2,148 @@
+            'missing Mod names nargs nzchar oldClass on.exit pos.to.env' +
+            'proc.time prod quote range Re rep retracemem return round' +
+            'seq_along seq_len seq.int sign signif sin sinh sinpi sqrt' +
+            'standardGeneric substitute sum switch tan tanh tanpi tracemem' +
+            'trigamma trunc unclass untracemem UseMethod xtfrm',
         },
         relevance: 0
       },
-      {
-        // hex value
-        className: 'number',
-        begin: "0[xX][0-9a-fA-F]+[Li]?\\b",
-        relevance: 0
-      },
-      {
-        // explicit integer
-        className: 'number',
-        begin: "\\d+(?:[eE][+\\-]?\\d*)?L\\b",
-        relevance: 0
-      },
-      {
-        // number with trailing decimal
-        className: 'number',
-        begin: "\\d+\\.(?!\\d)(?:i\\b)?",
-        relevance: 0
-      },
-      {
-        // number
-        className: 'number',
-        begin: "\\d+(?:\\.\\d*)?(?:[eE][+\\-]?\\d*)?i?\\b",
-        relevance: 0
-      },
-      {
-        // number with leading decimal
-        className: 'number',
-        begin: "\\.\\d+(?:[eE][+\\-]?\\d*)?i?\\b",
-        relevance: 0
-      },
+
+      // Roxygen comments
+      hljs.COMMENT(
+        /#'/,
+        /$/,
+        {
+          contains: [
+            {
+              // Handle `@examples` separately to cause all subsequent code
+              // until the next `@`-tag on its own line to be kept as-is,
+              // preventing highlighting. This code is example R code, so nested
+              // doctags shouldn’t be treated as such. See
+              // `test/markup/r/roxygen.txt` for an example.
+              className: 'doctag',
+              begin: '@examples',
+              starts: {
+                contains: [
+                  { begin: /\n/ },
+                  {
+                    begin: /#'\s*(?=@[a-zA-Z]+)/,
+                    endsParent: true,
+                  },
+                  {
+                    begin: /#'/,
+                    end: /$/,
+                    excludeBegin: true,
+                  }
+                ]
+              }
+            },
+            {
+              // Handle `@param` to highlight the parameter name following
+              // after.
+              className: 'doctag',
+              begin: '@param',
+              starts: {
+                end: /$/,
+                contains: [
+                  {
+                    className: 'meta',
+                    begin: IDENT_RE,
+                    endsParent: true
+                  }
+                ]
+              }
+            },
+            {
+              className: 'doctag',
+              begin: /@[a-zA-Z]+/
+            },
+            {
+              className: 'meta-keyword',
+              begin: /\\[a-zA-Z]+/,
+            }
+          ]
+        }
+      ),
+
+      hljs.HASH_COMMENT_MODE,
 
       {
-        // escaped identifier
-        begin: '`',
```

Comment:
Did you remove this on purpose?  It was probably a relevance boost... I'd consider bringing it back but using the 

```
`[IDENT]`
```

Pattern (with no className) so it can still serve as a relevance boost vs other langauges perhaps stealing these as "strings".  That's probably why it was there originally.

Reply:
The previous version actually set `relevance: 0` so I removed it because it was no longer necessary, after including it into `IDENT_RE` to make some other stuff work (see 89a346e3c83791a87655a10527241b3396c37883).

But I’ve now put it back in since it’s once again necessary after moving the keywords to the top level.